Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -252,12 +270,14 @@ class APIClient {
 
     func enterRecoveryMode() {
         // Pauses all the regular requests until recovery is completed.
+        log.debug("Entering recovery mode", subsystems: .offlineSupport)
         isRecoveryModeOn = true
         operationQueue.isSuspended = true
     }
 
     func exitRecoveryMode() {
         // Once recovery is done, regular requests can go through again.
+        log.debug("Leaving recovery mode", subsystems: .offlineSupport)
         isRecoveryModeOn = false
         operationQueue.isSuspended = false
```

Comment:
Before I forget, does this automatically kick off the OQ? From memory, it doesn't but I could be wrong...

Reply:
It is resuming the Operation Queue just by setting this value 😄

Reply:
`You may continue to add operations to a queue that is suspended but those operations are not scheduled for execution until you change this property to false.`

Reply:
No problem it was just one to check 💯